ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -49,14 +49,14 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-30/temp_commit:
+  fedora-30/test_user_permissions:
     requires: [fedora-30/build]
     priority: 50
     job:
       class: RunPytest
       args:
         build_url: '{fedora-30/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_user_permissions.py
```

Comment:
Here it will execute whole testsuite. Probably you might interested to execute only test you have written.
It can be done like:
test_integration/test_user_permissions.py::TestClass::test_case

Reply:
@mrizwan93, thanks noted this.